### DISCUSSION TITLE
[@types/node] Property 'prototype' is missing in type 'MyClass' but required in type 'EventTarget'.

### DISCUSSION THREAD

**Discussion by l0drex** [DefinitelyTyped/DefinitelyTyped#62761] [Issues with a @types package]
I would like to upgrade @types/node from 18.8.3 to 18.11.0, but with the new version a new issue showed up.
I am implementing EventTarget in one of my classes and have the functions `addEventListener` and `removeEventListener`. This worked before. Now I see the following errors:
> Property 'prototype' is missing in type 'MyClass' but required in type 'EventTarget'. 

When I add a new attribute `prototype`, I see this:
> Type 'MyClass' provides no match for the signature 'new (): EventTarget'.

This seems like a bug in the types to me.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
@l0drex thanks for your feedback! The issue is related to latest effort to 'unchain' Node types from DOM library, but it can be issue with TS, not related to types itself. For details see: #62759
